### DISCUSSION TITLE
[BUG] Old tray system icons window crash

### DISCUSSION THREAD

**Discussion by hiranokite** [valinet/ExplorerPatcher#1023] [General]
Taskbar style: windows 10

Can't change settings in "Taskbar > Customize system icons in the notification area" if "System tray > Show seconds in the clock" is enabled, the window just crash.

Tested on Windows 11.

**Comment by valinet** [comment]
Known for a very very very long time: https://github.com/valinet/ExplorerPatcher/issues/236

It's a File Explorer bug, happens on Windows 10 without ExplorerPatcher as well.

Also, it crashes without seconds enabled as well, when minutes change =))) Basically, it crashes when the clock updates and you're still in *that* particular window. Mind you, that actual window has been deprecated by Microsoft for quite some time, in latest Windows 10 builds they had a separate UI in Settings to toggle those icons (which didn't really work, neither it, nor this CPL).

**Reply by hiranokite** [reply]
ah, sorry, tried to find a listed issue specifically about "Show seconds in the clock" but no luck.

**Reply by valinet** [reply]
Yeah, no worries, I know about it, but don't really know how to solve it. You could rather say it's really a "known issue".
